sdk/search/search/review/search.api.md

```diff
@@ -374,6 +356,256 @@ export interface KeywordTokenizerV2 {
     odatatype: "#Microsoft.Azure.Search.KeywordTokenizerV2";
 }
 
+// @public
+export enum KnownAnalyzerNames {
+    // (undocumented)
+    Arlucene = "ar.lucene",
+    // (undocumented)
+    Armicrosoft = "ar.microsoft",
```

Comment:
I read this as "Arm icrosoft" 😀 Maybe make them PascalCase?

Reply:
yeah, went through and renamed these + added nice doc strings. Would be great to get those doc strings added to the swagger... ;)